flink-tests/src/test/java/org/apache/flink/test/scheduling/JMFailoverITCase.java

```diff
@@ -98,7 +98,7 @@
 import java.util.stream.IntStream;
 
 import static org.apache.flink.util.Preconditions.checkState;
-import static org.junit.Assert.assertEquals;
+import static org.assertj.core.api.Assertions.assertThat;
 
 /** ITCase for JM failover. */
 public class JMFailoverITCase {
```

Comment:
Junit5 should be used.

Reply:
This case utilizes the TestExecutorResource, which is compatible with JUnit 4.